docs/wiki/identity.md

```diff
@@ -18,7 +18,10 @@ me.tid
 # or
 j.core.identity.me.tid
 ```
-
+To register your identity in TFGrid
+```python
+me.register()
```

Comment:
isn't it `j.core.identity.me.register()`?

Reply:
```me = j.core.identity.new("name", "threebot_name", "threebot_mail", "words") ```